### PR TITLE
add ubuntu test cases on asus_kgpe_d16/dasharo-compatibility/os.md

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed off by Przemysław Banasiak [przemyslaw.banasiak@3mdeb.com]

### CODE REVIEW

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/os.md [hunk 1]
```diff
@@ -57,7 +57,67 @@ This test aims to verify that DUT logs properly into Debian booted from USB.
...
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `ubuntu login:`.
+6. Type proper login
+7. Wait for `Password:`
+8. Type the proper password
+9. Wait for `root@ubuntu:~#`
+
+**Expected result**
+
+The `root@ubuntu:~#` is visible after successful login.
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/os.md:123]: ``The `root@ubuntu:~#` prompt``

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `Debian GNU/Linux`
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:27]:
```suggestion
5. Wait for `Debian GNU/Linux`.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `debian login:`.
+6. Type proper login
+7. Wait for `Password:`
+8. Type the proper password
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:58]:
```suggestion
8. Type the proper password.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `debian login:`.
+6. Type proper login
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:56]:
```suggestion
6. Type proper login.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+**Expected result**
+
+The `root@debian:~#` prompt is visible after successful login.
+
+### LBT002.001 Ubuntu boot from USB
+
+**Test description**
+
+This test aims to verify that DUT boots properly Ubuntu from USB.
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:69]:
```suggestion
This test aims to verify whether the DUT boots properly Ubuntu from USB.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `Ubuntu GNU/Linux`
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:87]:
```suggestion
5. Wait for `Ubuntu GNU/Linux`.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `ubuntu login:`.
+6. Type proper login
+7. Wait for `Password:`
+8. Type the proper password
+9. Wait for `root@ubuntu:~#`
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:119]:
```suggestion
9. Wait for `root@ubuntu:~#`.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `ubuntu login:`.
+6. Type proper login
+7. Wait for `Password:`
+8. Type the proper password
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:118]:
```suggestion
8. Type the proper password.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `ubuntu login:`.
+6. Type proper login
+7. Wait for `Password:`
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:117]:
```suggestion
7. Wait for `Password:`.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the proper number for `USB_STICK` option.
+5. Wait for `ubuntu login:`.
+6. Type proper login
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:116]:
```suggestion
6. Type the proper login.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+**Expected result**
+
+The `Ubuntu GNU/Linux` is visible and confirms successful boot into OS.
+
+### LBT002.002 Log into booted Ubuntu
+
+**Test description**
+
+This test aims to verify that DUT logs properly into Ubuntu booted from USB.
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:97]:
```suggestion
This test aims to verify whether the DUT logs properly into Ubuntu booted from USB.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
+# Dasharo Compatibility: Debian Stable and Ubuntu LTS support
+
+## Test cases
+
+### LBT001.001 Debian boot from USB
+
+**Test description**
+
+This test aims to verify that DUT boots properly Debian from USB.
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:9]:
```suggestion
This test aims to verify whether the DUT boots properly Debian from USB.
```

File: docs/unified-test-documentation/dasharo-compatibility/os.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+
+**Expected result**
+
+The `Debian GNU/Linux` is visible and confirms successful boot into OS.
+
+### LBT001.002 Log into booted Debian
+
+**Test description**
+
+This test aims to verify that DUT logs properly into Debian booted from USB.
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-compatibility/os.md:37]:
```suggestion
This test aims to verify whether the DUT logs properly into Debian booted from USB.
```

**Comment by macpijan** [comment]
@sulewskiprzemyslaw What is the state of this PR?

**Comment by sulewskiprzemyslaw** [comment]
@macpijan @PrzemyslawBanasiak13 I decided to close this PR. This test suite has already been added to Dasharo test documentation. In my opinion, there is no need to add test cases describing booting from USB - we have `Generic test setup` section describing this operation among other things.

**Pull request closed without merging**